mmir-plugin-asr-bingWS-web/README.md

```diff
@@ -0,0 +1,62 @@
+# mmir-plugin-asr-nuance-web
+
+Cordova plugin for the MMIR framework that allows Automatic Speech Recognition (ASR) via Bing web services
+
+## configure CSP
```

Comment:
remove this section, or update, if there is specific CSP configuration that this plugin requires